metron-interface/metron-alerts/package.json

```diff
@@ -68,15 +65,14 @@
     "karma-jasmine-html-reporter": "^0.2.2",
     "node-sass": "^4.9.3",
     "npm-run-all": "^4.1.5",
-    "optimist": "0.6.1",
     "protractor": "^5.4.0",
     "protractor-flake": "^3.3.0",
     "rxjs-tslint": "^0.1.5",
     "serve-favicon": "^2.5.0",
-    "serve-static": "^1.13.2",
     "ssh2": "^0.5.5",
+    "start-server-and-test": "^1.7.11",
+    "ts-node": "^7.0.0",
     "tslint": "~5.0.0",
-    "typescript": "~2.9.2",
-    "yamljs": "^0.3.0"
+    "typescript": "~2.9.2"
```

Comment:
I was able to remove
- express
- http-proxy-middleware
- optimist
- serve-static
- yamljs

dependencies and added
- start-server-and-test